src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs

```diff
@@ -81,19 +81,22 @@ public void notebook_metadata_default_language_is_honored_in_cells_without_langu
                 });
     }
 
-    [Fact]
-    public void Default_language_can_be_specified_in_metadata()
+    [Theory]
+    [InlineData("C#", "csharp")]
+    [InlineData("F#", "fsharp")]
+    [InlineData("f#", "fsharp")]
+    [InlineData("PowerShell", "powershell")]
```

Comment:
isn't this kernel name `pwsh` and `powershell` an alias?

Reply:
Here's what we have in the kernelspec: 
![image](https://user-images.githubusercontent.com/547415/196792843-235e4350-7a24-47a9-a6f7-161072717d6a.png)